CMakeLists.txt

```diff
@@ -81,11 +81,8 @@ tristate_feature_auto(ENABLE_RIPEMD160 "Enable RIPEMD-160 hash support.")
 
 option(ENABLE_CRYPTO_REFRESH "Enable crypto-refresh support (v6)")
 option(ENABLE_PQC "Enable PQC support - requires ENABLE_CRYPTO_REFRESH")
-
-if((NOT ENABLE_CRYPTO_REFRESH) AND ENABLE_PQC)
-  message(FATAL_ERROR "ENABLE_PQC requires ENABLE_CRYPTO_REFRESH")
-endif()
-
+option(ENABLE_PQC_MLKEM_IPD "Use ML-KEM-ipd instead of Kyber")
+option(ENABLE_PQC_DBG_LOG "If enabled, logs intermediate values of computations. Used for debugging, do not enable for productive use.")
```

Comment:
As I understand at some point it time this would be no longer needed and would be deleted from the codebase?